opt/Makefile

```diff
@@ -219,9 +219,13 @@ pack: $(INSTALLED_TARGET)
 	$(SHOW)find $(INSTALL_DIR) -name "*.so" -exec chmod +x {} \;
 	$(SHOW)mkdir -p $(ROOT)/bin/artifacts
 ifneq ($(PACK_DEPS),0)
-	$(SHOW)DEVICE=$(DEVICE) BINDIR=$(ROOT)/bin/artifacts INSTALL_DIR=$(INSTALL_DIR) BRANCH=$(BRANCH) RAMP=1 DEPS=1 ./pack.sh
+	$(SHOW)DEVICE=$(DEVICE) BINDIR=$(ROOT)/bin/artifacts INSTALL_DIR=$(INSTALL_DIR) BRANCH=$(BRANCH) RAMP=0 DEPS=1 ./pack.sh
```

Comment:
why `RAMP=0` here?
Also, we can avoid the large `ifneq ($(PACK_DEPS),0)` and just arrange `DEPS=$(PACK_DEPS)` with the following:
```
ifneq ($(PACK_DEPS),0)
override PACK_DEPS:=1
endif
```